src/templates/naive.js

```diff
@@ -100,11 +100,11 @@ function base() {
 
     function $setter(func, ...args) {
       if ($inBatch || $inRecalculate || $batchingStrategy) {
+        $batchPending.push({ func, args });
```

Comment:
`$batchPending.push()` is now before `$batchingStrategy.call()` for synchronous batching strategy to execute the setter.